test/xpu_api/numerics/c.math/cmath.pass.cpp

```diff
@@ -35,15 +35,19 @@ template <typename HasDoubleSupportInRuntime, typename HasLongDoubleSupportInCom
 void test_nearbyint()
 {
     static_assert((std::is_same<decltype(dpl::nearbyint((float)0)), float>::value), "");
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((bool)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((unsigned short)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((int)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((unsigned int)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((long)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((unsigned long)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((long long)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((unsigned long long)0)), double>::value), ""))
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::nearbyint((double)0)), double>::value), ""))
+    auto fnc = []()
+    {
+        static_assert((std::is_same<decltype(dpl::nearbyint((bool)0)), double>::value), "");
```

Comment:
that line number and formatting does match to the corresponded lines an original LLVM test?